charts/rancher-monitoring/v0.2.0/questions.yml

```diff
@@ -1 +1 @@
-rancher_min_version: 2.4.9-rc1
+rancher_min_version: 2.5.4-rc1
```

Comment:
Does this need to be 2.5.3-rc1 for QA to deploy it onto a 2.5.4 cluster? or is this okay?

Reply:
Nope, 2.5.4-rc1 should work